developer/webservices/resource-mapping-schema.md

```diff
@@ -1,199 +1,202 @@
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+    </xs:all>
+  </xs:complexType>
+  <xs:complexType name="FieldParameterMapType">
+    <xs:sequence>
+      <xs:element name="Field" minOccurs="0" maxOccurs="unbounded">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="FieldName" type="xs:string" />
+            <xs:element name="ParameterName" type="xs:string" />
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+  <xs:complexType name="ParameterSetsType">
+    <xs:sequence>
+      <xs:element name="ParameterSet" minOccurs="0" maxOccurs="unbounded">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="Name" type="xs:string" />
+            <xs:element name="Parameter" maxOccurs="unbounded">
+              <xs:complexType>
+                <xs:all>
+                  <xs:element name="Name" type="xs:string" />
+                  <xs:element name="Type" type="xs:string" minOccurs="0" />
+                  <xs:element name="IsSwitch" type="xs:string" minOccurs="0" />
+                  <xs:element name="IsMandatory" type="xs:string" minOccurs="0" />
+                </xs:all>
+              </xs:complexType>
+            </xs:element>
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+  <xs:complexType name="ParameterValuesType">
+    <xs:sequence>
+      <xs:element name="ParameterValue" minOccurs="0" maxOccurs="unbounded">
+        <xs:complexType>
+          <xs:all>
+            <xs:element name="ParameterName" type="xs:string" />
+            <xs:element name="Value" type="xs:string" />
+          </xs:all>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+  <xs:complexType name="OptionsType">
+    <xs:sequence>
+      <xs:element name="ParameterName" type="xs:string" minOccurs="0" maxOccurs="unbounded" />
+    </xs:sequence>
+  </xs:complexType>
+  <xs:complexType name="AssociationFieldType">
+    <xs:all>
+      <xs:element name="Name" type="xs:string" />
+      <xs:element name="GetReference" type="GetAssociationReferenceType" minOccurs="0" />
+      <xs:element name="AddReference" type="AssociationReferenceType" />
+      <xs:element name="RemoveReference" type="AssociationReferenceType" />
+    </xs:all>
+  </xs:complexType>
+  <xs:complexType name="AssociationReferenceType">
+    <xs:all>
+      <xs:element name="Cmdlet" type="xs:string" />
+      <xs:element name="ParameterForThisObject" type="xs:string" />
+      <xs:element name="ParameterForReferredObject" type="xs:string" />
+    </xs:all>
+  </xs:complexType>
+  <xs:complexType name="GetAssociationReferenceType">
+    <xs:all>
+      <xs:element name="Cmdlet" type="xs:string" />
+      <xs:element name="ParameterForThisObject" type="xs:string" />
+    </xs:all>
+  </xs:complexType>
+</xs:schema>
+```


```

Comment:
Control character

Reply:
This was an issue with my code fence logic and is fixed